build/generate-svg.js

```diff
@@ -50,7 +50,8 @@ async function processFile(file, {prefix = ''} = {}) {
 async function main() {
   try {
     await mkdir(outputDir);
-  } catch {}
+  } catch {
+  }
```

Comment:
Is it really an issue? eslint certainly doesn't complain about it.

Reply:
it's not - the importand thing for me was about markdown and ini files :)